tests/qp/quantile_pdf_constructors/test_cdf_spline_derivative.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+import logging
+
+import numpy as np
+
+import qp
+from qp.quantile_pdf_constructors import AbstractQuantilePdfConstructor, \
+    CdfSplineDerivative
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
This logger doesn't get used. Is it just instantiated so there's some root logging handler?

Reply:
I usually add the LOGGER as boilerplate and use it eventually, but I didn't this time around. I'll pull these out.